dynamic:
  radius_x: 1.5
  radius_y: 1.5
  linear_speed: 0.25
initial_point:
  num: 6
  name: circle
  ReadyPoint_1: 0
  ReadyPoint_2: 62
  ReadyPoint_3: 124
  ReadyPoint_4: 186
  ReadyPoint_5: 248
  ReadyPoint_6: 310
point_1: [1.5, 0, 0]
point_2: [1.499791, 0.02506473, 0]
point_3: [1.499162, 0.05012247, 0]
point_4: [1.498116, 0.0751662, 0]
point_5: [1.49665, 0.100189, 0]
point_6: [1.494767, 0.1251837, 0]
point_7: [1.492467, 0.1501435, 0]
point_8: [1.489749, 0.1750614, 0]
point_9: [1.486616, 0.1999304, 0]
point_10: [1.483068, 0.2247436, 0]
point_11: [1.479105, 0.249494, 0]
point_12: [1.47473, 0.2741748, 0]
point_13: [1.469943, 0.298779, 0]
point_14: [1.464745, 0.3232998, 0]
point_15: [1.459138, 0.3477302, 0]
point_16: [1.453124, 0.3720636, 0]
point_17: [1.446704, 0.3962931, 0]
point_18: [1.43988, 0.4204119, 0]
point_19: [1.432654, 0.4444134, 0]
point_20: [1.425028, 0.4682907, 0]
point_21: [1.417004, 0.4920373, 0]
point_22: [1.408584, 0.5156465, 0]
point_23: [1.399771, 0.5391117, 0]
point_24: [1.390567, 0.5624263, 0]
point_25: [1.380975, 0.5855839, 0]
point_26: [1.370997, 0.608578, 0]
point_27: [1.360636, 0.6314021, 0]
point_28: [1.349896, 0.65405, 0]
point_29: [1.338778, 0.6765152, 0]
point_30: [1.327287, 0.6987915, 0]
point_31: [1.315425, 0.7208726, 0]
point_32: [1.303196, 0.7427525, 0]
point_33: [1.290602, 0.7644249, 0]
point_34: [1.277649, 0.7858839, 0]
point_35: [1.264338, 0.8071235, 0]
point_36: [1.250675, 0.8281376, 0]
point_37: [1.236662, 0.8489207, 0]
point_38: [1.222304, 0.8694665, 0]
point_39: [1.207605, 0.8897697, 0]
point_40: [1.192569, 0.9098243, 0]
point_41: [1.177199, 0.9296249, 0]
point_42: [1.161501, 0.9491658, 0]
point_43: [1.145478, 0.9684418, 0]
point_44: [1.129136, 0.9874473, 0]
point_45: [1.112478, 1.006177, 0]
point_46: [1.09551, 1.024626, 0]
point_47: [1.078236, 1.042789, 0]
point_48: [1.06066, 1.06066, 0]
point_49: [1.042789, 1.078236, 0]
point_50: [1.024626, 1.09551, 0]
point_51: [1.006177, 1.112478, 0]
point_52: [0.9874473, 1.129136, 0]
point_53: [0.9684418, 1.145478, 0]
point_54: [0.9491658, 1.161501, 0]
point_55: [0.9296249, 1.177199, 0]
point_56: [0.9098243, 1.192569, 0]
point_57: [0.8897697, 1.207605, 0]
point_58: [0.8694665, 1.222304, 0]
point_59: [0.8489207, 1.236662, 0]
point_60: [0.8281376, 1.250675, 0]
point_61: [0.8071235, 1.264338, 0]
point_62: [0.7858839, 1.277649, 0]
point_63: [0.7644249, 1.290602, 0]
point_64: [0.7427525, 1.303196, 0]
point_65: [0.7208726, 1.315425, 0]
point_66: [0.6987915, 1.327287, 0]
point_67: [0.6765152, 1.338778, 0]
point_68: [0.65405, 1.349896, 0]
point_69: [0.6314021, 1.360636, 0]
point_70: [0.608578, 1.370997, 0]
point_71: [0.5855839, 1.380975, 0]
point_72: [0.5624263, 1.390567, 0]
point_73: [0.5391117, 1.399771, 0]
point_74: [0.5156465, 1.408584, 0]
point_75: [0.4920373, 1.417004, 0]
point_76: [0.4682907, 1.425028, 0]
point_77: [0.4444134, 1.432654, 0]
point_78: [0.4204119, 1.43988, 0]
point_79: [0.3962931, 1.446704, 0]
point_80: [0.3720636, 1.453124, 0]
point_81: [0.3477302, 1.459138, 0]
point_82: [0.3232998, 1.464745, 0]
point_83: [0.298779, 1.469943, 0]
point_84: [0.2741748, 1.47473, 0]
point_85: [0.249494, 1.479105, 0]
point_86: [0.2247436, 1.483068, 0]
point_87: [0.1999304, 1.486616, 0]
point_88: [0.1750614, 1.489749, 0]
point_89: [0.1501435, 1.492467, 0]
point_90: [0.1251837, 1.494767, 0]
point_91: [0.100189, 1.49665, 0]
point_92: [0.0751662, 1.498116, 0]
point_93: [0.05012247, 1.499162, 0]
point_94: [0.02506473, 1.499791, 0]
point_95: [1.83697e-16, 1.5, 0]
point_96: [-0.02506473, 1.499791, 0]
point_97: [-0.05012247, 1.499162, 0]
point_98: [-0.0751662, 1.498116, 0]
point_99: [-0.100189, 1.49665, 0]
point_100: [-0.1251837, 1.494767, 0]
point_101: [-0.1501435, 1.492467, 0]
point_102: [-0.1750614, 1.489749, 0]
point_103: [-0.1999304, 1.486616, 0]
point_104: [-0.2247436, 1.483068, 0]
point_105: [-0.249494, 1.479105, 0]
point_106: [-0.2741748, 1.47473, 0]
point_107: [-0.298779, 1.469943, 0]
point_108: [-0.3232998, 1.464745, 0]
point_109: [-0.3477302, 1.459138, 0]
point_110: [-0.3720636, 1.453124, 0]
point_111: [-0.3962931, 1.446704, 0]
point_112: [-0.4204119, 1.43988, 0]
point_113: [-0.4444134, 1.432654, 0]
point_114: [-0.4682907, 1.425028, 0]
point_115: [-0.4920373, 1.417004, 0]
point_116: [-0.5156465, 1.408584, 0]
point_117: [-0.5391117, 1.399771, 0]
point_118: [-0.5624263, 1.390567, 0]
point_119: [-0.5855839, 1.380975, 0]
point_120: [-0.608578, 1.370997, 0]
point_121: [-0.6314021, 1.360636, 0]
point_122: [-0.65405, 1.349896, 0]
point_123: [-0.6765152, 1.338778, 0]
point_124: [-0.6987915, 1.327287, 0]
point_125: [-0.7208726, 1.315425, 0]
point_126: [-0.7427525, 1.303196, 0]
point_127: [-0.7644249, 1.290602, 0]
point_128: [-0.7858839, 1.277649, 0]
point_129: [-0.8071235, 1.264338, 0]
point_130: [-0.8281376, 1.250675, 0]
point_131: [-0.8489207, 1.236662, 0]
point_132: [-0.8694665, 1.222304, 0]
point_133: [-0.8897697, 1.207605, 0]
point_134: [-0.9098243, 1.192569, 0]
point_135: [-0.9296249, 1.177199, 0]
point_136: [-0.9491658, 1.161501, 0]
point_137: [-0.9684418, 1.145478, 0]
point_138: [-0.9874473, 1.129136, 0]
point_139: [-1.006177, 1.112478, 0]
point_140: [-1.024626, 1.09551, 0]
point_141: [-1.042789, 1.078236, 0]
point_142: [-1.06066, 1.06066, 0]
point_143: [-1.078236, 1.042789, 0]
point_144: [-1.09551, 1.024626, 0]
point_145: [-1.112478, 1.006177, 0]
point_146: [-1.129136, 0.9874473, 0]
point_147: [-1.145478, 0.9684418, 0]
point_148: [-1.161501, 0.9491658, 0]
point_149: [-1.177199, 0.9296249, 0]
point_150: [-1.192569, 0.9098243, 0]
point_151: [-1.207605, 0.8897697, 0]
point_152: [-1.222304, 0.8694665, 0]
point_153: [-1.236662, 0.8489207, 0]
point_154: [-1.250675, 0.8281376, 0]
point_155: [-1.264338, 0.8071235, 0]
point_156: [-1.277649, 0.7858839, 0]
point_157: [-1.290602, 0.7644249, 0]
point_158: [-1.303196, 0.7427525, 0]
point_159: [-1.315425, 0.7208726, 0]
point_160: [-1.327287, 0.6987915, 0]
point_161: [-1.338778, 0.6765152, 0]
point_162: [-1.349896, 0.65405, 0]
point_163: [-1.360636, 0.6314021, 0]
point_164: [-1.370997, 0.608578, 0]
point_165: [-1.380975, 0.5855839, 0]
point_166: [-1.390567, 0.5624263, 0]
point_167: [-1.399771, 0.5391117, 0]
point_168: [-1.408584, 0.5156465, 0]
point_169: [-1.417004, 0.4920373, 0]
point_170: [-1.425028, 0.4682907, 0]
point_171: [-1.432654, 0.4444134, 0]
point_172: [-1.43988, 0.4204119, 0]
point_173: [-1.446704, 0.3962931, 0]
point_174: [-1.453124, 0.3720636, 0]
point_175: [-1.459138, 0.3477302, 0]
point_176: [-1.464745, 0.3232998, 0]
point_177: [-1.469943, 0.298779, 0]
point_178: [-1.47473, 0.2741748, 0]
point_179: [-1.479105, 0.249494, 0]
point_180: [-1.483068, 0.2247436, 0]
point_181: [-1.486616, 0.1999304, 0]
point_182: [-1.489749, 0.1750614, 0]
point_183: [-1.492467, 0.1501435, 0]
point_184: [-1.494767, 0.1251837, 0]
point_185: [-1.49665, 0.100189, 0]
point_186: [-1.498116, 0.0751662, 0]
point_187: [-1.499162, 0.05012247, 0]
point_188: [-1.499791, 0.02506473, 0]
point_189: [-1.5, -4.824368e-16, 0]
point_190: [-1.499791, -0.02506473, 0]
point_191: [-1.499162, -0.05012247, 0]
point_192: [-1.498116, -0.0751662, 0]
point_193: [-1.49665, -0.100189, 0]
point_194: [-1.494767, -0.1251837, 0]
point_195: [-1.492467, -0.1501435, 0]
point_196: [-1.489749, -0.1750614, 0]
point_197: [-1.486616, -0.1999304, 0]
point_198: [-1.483068, -0.2247436, 0]
point_199: [-1.479105, -0.249494, 0]
point_200: [-1.47473, -0.2741748, 0]
point_201: [-1.469943, -0.298779, 0]
point_202: [-1.464745, -0.3232998, 0]
point_203: [-1.459138, -0.3477302, 0]
point_204: [-1.453124, -0.3720636, 0]
point_205: [-1.446704, -0.3962931, 0]
point_206: [-1.43988, -0.4204119, 0]
point_207: [-1.432654, -0.4444134, 0]
point_208: [-1.425028, -0.4682907, 0]
point_209: [-1.417004, -0.4920373, 0]
point_210: [-1.408584, -0.5156465, 0]
point_211: [-1.399771, -0.5391117, 0]
point_212: [-1.390567, -0.5624263, 0]
point_213: [-1.380975, -0.5855839, 0]
point_214: [-1.370997, -0.608578, 0]
point_215: [-1.360636, -0.6314021, 0]
point_216: [-1.349896, -0.65405, 0]
point_217: [-1.338778, -0.6765152, 0]
point_218: [-1.327287, -0.6987915, 0]
point_219: [-1.315425, -0.7208726, 0]
point_220: [-1.303196, -0.7427525, 0]
point_221: [-1.290602, -0.7644249, 0]
point_222: [-1.277649, -0.7858839, 0]
point_223: [-1.264338, -0.8071235, 0]
point_224: [-1.250675, -0.8281376, 0]
point_225: [-1.236662, -0.8489207, 0]
point_226: [-1.222304, -0.8694665, 0]
point_227: [-1.207605, -0.8897697, 0]
point_228: [-1.192569, -0.9098243, 0]
point_229: [-1.177199, -0.9296249, 0]
point_230: [-1.161501, -0.9491658, 0]
point_231: [-1.145478, -0.9684418, 0]
point_232: [-1.129136, -0.9874473, 0]
point_233: [-1.112478, -1.006177, 0]
point_234: [-1.09551, -1.024626, 0]
point_235: [-1.078236, -1.042789, 0]
point_236: [-1.06066, -1.06066, 0]
point_237: [-1.042789, -1.078236, 0]
point_238: [-1.024626, -1.09551, 0]
point_239: [-1.006177, -1.112478, 0]
point_240: [-0.9874473, -1.129136, 0]
point_241: [-0.9684418, -1.145478, 0]
point_242: [-0.9491658, -1.161501, 0]
point_243: [-0.9296249, -1.177199, 0]
point_244: [-0.9098243, -1.192569, 0]
point_245: [-0.8897697, -1.207605, 0]
point_246: [-0.8694665, -1.222304, 0]
point_247: [-0.8489207, -1.236662, 0]
point_248: [-0.8281376, -1.250675, 0]
point_249: [-0.8071235, -1.264338, 0]
point_250: [-0.7858839, -1.277649, 0]
point_251: [-0.7644249, -1.290602, 0]
point_252: [-0.7427525, -1.303196, 0]
point_253: [-0.7208726, -1.315425, 0]
point_254: [-0.6987915, -1.327287, 0]
point_255: [-0.6765152, -1.338778, 0]
point_256: [-0.65405, -1.349896, 0]
point_257: [-0.6314021, -1.360636, 0]
point_258: [-0.608578, -1.370997, 0]
point_259: [-0.5855839, -1.380975, 0]
point_260: [-0.5624263, -1.390567, 0]
point_261: [-0.5391117, -1.399771, 0]
point_262: [-0.5156465, -1.408584, 0]
point_263: [-0.4920373, -1.417004, 0]
point_264: [-0.4682907, -1.425028, 0]
point_265: [-0.4444134, -1.432654, 0]
point_266: [-0.4204119, -1.43988, 0]
point_267: [-0.3962931, -1.446704, 0]
point_268: [-0.3720636, -1.453124, 0]
point_269: [-0.3477302, -1.459138, 0]
point_270: [-0.3232998, -1.464745, 0]
point_271: [-0.298779, -1.469943, 0]
point_272: [-0.2741748, -1.47473, 0]
point_273: [-0.249494, -1.479105, 0]
point_274: [-0.2247436, -1.483068, 0]
point_275: [-0.1999304, -1.486616, 0]
point_276: [-0.1750614, -1.489749, 0]
point_277: [-0.1501435, -1.492467, 0]
point_278: [-0.1251837, -1.494767, 0]
point_279: [-0.100189, -1.49665, 0]
point_280: [-0.0751662, -1.498116, 0]
point_281: [-0.05012247, -1.499162, 0]
point_282: [-0.02506473, -1.499791, 0]
point_283: [-3.673941e-16, -1.5, 0]
point_284: [0.02506473, -1.499791, 0]
point_285: [0.05012247, -1.499162, 0]
point_286: [0.0751662, -1.498116, 0]
point_287: [0.100189, -1.49665, 0]
point_288: [0.1251837, -1.494767, 0]
point_289: [0.1501435, -1.492467, 0]
point_290: [0.1750614, -1.489749, 0]
point_291: [0.1999304, -1.486616, 0]
point_292: [0.2247436, -1.483068, 0]
point_293: [0.249494, -1.479105, 0]
point_294: [0.2741748, -1.47473, 0]
point_295: [0.298779, -1.469943, 0]
point_296: [0.3232998, -1.464745, 0]
point_297: [0.3477302, -1.459138, 0]
point_298: [0.3720636, -1.453124, 0]
point_299: [0.3962931, -1.446704, 0]
point_300: [0.4204119, -1.43988, 0]
point_301: [0.4444134, -1.432654, 0]
point_302: [0.4682907, -1.425028, 0]
point_303: [0.4920373, -1.417004, 0]
point_304: [0.5156465, -1.408584, 0]
point_305: [0.5391117, -1.399771, 0]
point_306: [0.5624263, -1.390567, 0]
point_307: [0.5855839, -1.380975, 0]
point_308: [0.608578, -1.370997, 0]
point_309: [0.6314021, -1.360636, 0]
point_310: [0.65405, -1.349896, 0]
point_311: [0.6765152, -1.338778, 0]
point_312: [0.6987915, -1.327287, 0]
point_313: [0.7208726, -1.315425, 0]
point_314: [0.7427525, -1.303196, 0]
point_315: [0.7644249, -1.290602, 0]
point_316: [0.7858839, -1.277649, 0]
point_317: [0.8071235, -1.264338, 0]
point_318: [0.8281376, -1.250675, 0]
point_319: [0.8489207, -1.236662, 0]
point_320: [0.8694665, -1.222304, 0]
point_321: [0.8897697, -1.207605, 0]
point_322: [0.9098243, -1.192569, 0]
point_323: [0.9296249, -1.177199, 0]
point_324: [0.9491658, -1.161501, 0]
point_325: [0.9684418, -1.145478, 0]
point_326: [0.9874473, -1.129136, 0]
point_327: [1.006177, -1.112478, 0]
point_328: [1.024626, -1.09551, 0]
point_329: [1.042789, -1.078236, 0]
point_330: [1.06066, -1.06066, 0]
point_331: [1.078236, -1.042789, 0]
point_332: [1.09551, -1.024626, 0]
point_333: [1.112478, -1.006177, 0]
point_334: [1.129136, -0.9874473, 0]
point_335: [1.145478, -0.9684418, 0]
point_336: [1.161501, -0.9491658, 0]
point_337: [1.177199, -0.9296249, 0]
point_338: [1.192569, -0.9098243, 0]
point_339: [1.207605, -0.8897697, 0]
point_340: [1.222304, -0.8694665, 0]
point_341: [1.236662, -0.8489207, 0]
point_342: [1.250675, -0.8281376, 0]
point_343: [1.264338, -0.8071235, 0]
point_344: [1.277649, -0.7858839, 0]
point_345: [1.290602, -0.7644249, 0]
point_346: [1.303196, -0.7427525, 0]
point_347: [1.315425, -0.7208726, 0]
point_348: [1.327287, -0.6987915, 0]
point_349: [1.338778, -0.6765152, 0]
point_350: [1.349896, -0.65405, 0]
point_351: [1.360636, -0.6314021, 0]
point_352: [1.370997, -0.608578, 0]
point_353: [1.380975, -0.5855839, 0]
point_354: [1.390567, -0.5624263, 0]
point_355: [1.399771, -0.5391117, 0]
point_356: [1.408584, -0.5156465, 0]
point_357: [1.417004, -0.4920373, 0]
point_358: [1.425028, -0.4682907, 0]
point_359: [1.432654, -0.4444134, 0]
point_360: [1.43988, -0.4204119, 0]
point_361: [1.446704, -0.3962931, 0]
point_362: [1.453124, -0.3720636, 0]
point_363: [1.459138, -0.3477302, 0]
point_364: [1.464745, -0.3232998, 0]
point_365: [1.469943, -0.298779, 0]
point_366: [1.47473, -0.2741748, 0]
point_367: [1.479105, -0.249494, 0]
point_368: [1.483068, -0.2247436, 0]
point_369: [1.486616, -0.1999304, 0]
point_370: [1.489749, -0.1750614, 0]
point_371: [1.492467, -0.1501435, 0]
point_372: [1.494767, -0.1251837, 0]
point_373: [1.49665, -0.100189, 0]
point_374: [1.498116, -0.0751662, 0]
point_375: [1.499162, -0.05012247, 0]
point_376: [1.499791, -0.02506473, 0]